Classes/UT3LeviathanShieldEffectRed.uc

```diff
@@ -18,7 +18,8 @@ DefaultProperties
         CoordinateSystem=PTCS_Relative
         MaxParticles=1
         UniformSize=false
-        StartSizeRange=(X=(Min=0.6,Max=0.6),Y=(Min=0.75,Max=0.75),Z=(Min=1.0,Max=1.0))
+        StartSizeRange=(X=(Min=1.0,Max=1.0),Y=(Min=1.0,Max=1.0),Z=(Min=1.0,Max=1.0))
+        //StartSizeRange=(X=(Min=0.7,Max=0.7),Y=(Min=1.4,Max=1.4),Z=(Min=1.6,Max=1.6))
```

Comment:
No need to keep the previous testing values around as comments.

Reply:
I'd like to keep that as to me those values are more correct, the 1.0 just seems small but that's just my opinion and possibly an optical illusion as I will say on the hostile side of the Levi shield it doesn't look as big for some reason as it does when you're driving.....odd I agree